web/models/stable_diffusion/main.py

```diff
@@ -72,8 +72,8 @@ def get_models():
         IREE_EXTRA_ARGS += [
             "--iree-flow-enable-conv-nchw-to-nhwc-transform",
             "--iree-flow-enable-padding-linalg-ops",
-            "--iree-flow-linalg-ops-padding-size=16",
-            "--iree-flow-enable-iterator-space-fusion",
+            "--iree-flow-linalg-ops-padding-size=32",
+            "--iree-spirv-unify-aliased-resources=false",
```

Comment:
The most recent update to shark has removed this flag, so we can drop it from here